projects/mercury/src/dashboard/DashboardPage.js

```diff
@@ -0,0 +1,76 @@
+import React, {useContext} from "react";
+
+import withStyles from "@mui/styles/withStyles";
+import {Grid, Link, Paper, Typography} from "@mui/material";
+import BreadCrumbs from "../common/components/BreadCrumbs";
+import styles from "./DashboardPage.styles";
+import MetadataViewContext from "../metadata/views/MetadataViewContext";
+import ExternalMetadataSourceContext from "../metadata/external-sources/ExternalMetadataSourceContext";
+import UserContext from "../users/UserContext";
+import DomainInfo from "./DomainInfo";
+import {APPLICATION_NAME, METADATA_VIEW_MENU_LABEL} from "../constants";
+
+const DashboardPage = (props) => {
+    const {currentUser, classes} = props;
+    const {views} = useContext(MetadataViewContext);
+    const {externalMetadataSources} = useContext(ExternalMetadataSourceContext);
+    const canViewMetadata = currentUser && currentUser.canViewPublicMetadata && views && views.length > 0;
+
+    return (
+        <Grid container justifyContent="center" spacing="5">
```

Comment:
this whole content is visible even if not logged in (or backend is down).
Not sure if this is a good idea. I would distinguish those states and if backend is down or not logged in, show something else (e.g. info to log in to see the whole content or something like that).

Reply:
Not sure how to solve, let's discuss